assets/css/_comments.scss

```diff
@@ -35,11 +34,10 @@
 
 .comment {
   @include padding($base-spacing null);
-  border-bottom: $base-border;
+  border-top: $base-border;
```

Comment:
Can the `&last-child` `border-bottom: none;` below (line 44) be removed now that this has changed?